### PR TITLE
Add building icon

### PR DESCRIPTION
**Pull request by delnyn**
The  building icon. *(cf #78)*

### CODE REVIEW

File: building.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+<path d="M4 4C4 2.89543 4.79594 2 5.77778 2H18.2222C19.2041 2 20 2.89543 20 4V20C20 20.5304 19.8127 21.0391 19.4793 21.4142C19.1459 21.7893 18.6937 22 18.2222 22H5.77778C5.30628 22 4.8541 21.7893 4.5207 21.4142C4.1873 21.0391 4 20.5304 4 20V4Z" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (ericfennis) [inline building.svg:12]:
```suggestion
  <rect x="4" y="2" width="16" height="20" rx="2" ry="2" />
```

File: building.svg [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<path d="M9 22V18H15V22" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M8 6H8.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M16 6H16.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M12 6H12.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M12 10H12.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M12 14H12.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M16 10H16.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M16 14H16.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M8 10H8.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
+<path d="M8 14H8.01" stroke="black" stroke-width="2" stroke-linecap="round"/>
```
Comment (ericfennis) [inline building.svg:22]:
```suggestion
  <path d="M9 22v-4h6v4" />
  <path d="M8 6h.01" />
  <path d="M16 6h.01" />
  <path d="M12 6h.01" />
  <path d="M12 10h.01" />
  <path d="M12 14h.01" />
  <path d="M16 10h.01" />
  <path d="M16 14h.01" />
  <path d="M8 10h.01" />
  <path d="M8 14h.01" />
```

File: tags.json [hunk 3]
```diff
@@ -25,6 +25,7 @@
   "bookmark": ["read", "clip", "marker", "tag"],
   "box": ["cube"],
   "briefcase": ["work", "bag", "baggage", "folder"],
+  "building": ["organisations"],
```
Comment (ericfennis) [inline tags.json:28]: I prefer singular, because the object is singular as well.
```suggestion
  "building": ["organisation"],
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
🚀 Nice work @delnyn .

I have some small code improvements.

**Comment by delnyn** [comment]
OK,  commited every changes 👍

**Comment by ericfennis** [comment]
@delnyn Awesome work!

**Comment by locness3** [comment]
Nice work!

**Comment by delnyn** [comment]
Oops! Something is wrong, the icon is directly in the root of the repo 😓 sorry, I'm going to fix that...

**Pull request merged**